### PR TITLE
RUN-4088 persistent zoom level

### PR DESCRIPTION
**Pull request by licui3936**
This PR is persistent zoom level for each window. 
 [Add a new test for this feature ](https://testing-dashboard.openfin.co/#/app/tests/5b6b52a254b21953031f3911/edit)

Integration Tests:
[Windows 10](https://testing-dashboard.openfin.co/#/app/sessions/api/completed/5b6c59d954b21953031f3921)
[Windows 7](https://testing-dashboard.openfin.co/#/app/sessions/api/completed/5b6c587454b21953031f3920)

### CODE REVIEW

File: src/browser/api/window.js [hunk 1]
```diff
@@ -2409,6 +2412,13 @@ function restoreWindowPosition(identity, cb) {
                 break;
         }
 
+        // set zoom level
+        let level = savedBounds.zoomLevel;
+        if (level) {
```
Comment (HarsimranSingh) [inline src/browser/api/window.js]: What about the case where `level` is 0?

Comment (licui3936) [reply src/browser/api/window.js]: That's default value. So no need to call the function.

Comment (HarsimranSingh) [reply src/browser/api/window.js]: Oh yea you're right. Windows by default start at 0.

**Comment by rdepena** [comment]
🌳 Until the .34 staging cut.

File: src/browser/api/window.js [hunk 2]
```diff
@@ -2409,6 +2412,13 @@ function restoreWindowPosition(identity, cb) {
                 break;
         }
 
+        // set zoom level
+        let level = savedBounds.zoomLevel;
```
Comment (whyn07m3) [inline src/browser/api/window.js]: :neckbeard: how about `const { zoomLevel } = savedBounds;`?

File: src/browser/api/window.js [hunk 3]
```diff
@@ -2409,6 +2412,13 @@ function restoreWindowPosition(identity, cb) {
...
+        if (level) {
+            let browserWindow = getElectronBrowserWindow(identity);
```
Comment (whyn07m3) [inline src/browser/api/window.js]: :neckbeard: `const`?

**Comment by licui3936** [comment]
@whyn07m3  updated it. Thanks!

**Pull request merged**